tutorial/lesson_16_rgb_run.cpp

```diff
@@ -23,6 +23,14 @@
 
 #include "halide_benchmark.h"
 
+#define check_timing(faster, slower) \
```

Comment:
The advantage of the macro is that you get the variable names in the printf, but I think we should avoid it in a tutorial. People might be thrown by the stringification of arguments and think they're not understanding something important.

There's a printf just before each assert already, so how about just:
```
void check_timing(double faster, double slower) {
  if (faster < slower) {
    fprintf(stderr, "Warning: performance was worse than expected. %f should be less than %f\n", faster, slower);
  }
}
```

Reply:
Done